omero/sysadmins/version-requirements.txt

```diff
@@ -0,0 +1,1606 @@
+builds in place; no 2012 builds or testing in place yet.  No regular
+Windows CI deployment and testing in place other than manually.  Some
+manual testing on Windows 7 VMs and machines.
+
+UNIX (MacOS X)
+^^^^^^^^^^^^^^
+
+`General overview <http://en.wikipedia.org/wiki/OS_X>`__
+
+.. tabularcolumns:: |l|J|J|l|l|l|
+
+.. list-table::
+    :header-rows: 1
+
+    * - Version
+      - Release date
+      - Upstream support
+      - Homebrew support
+      - OME (build)
+      - OME (deploy)
+    * - 10.6
+      - from Jun 2008
+      - Ended
+      - Ended
+      - Dropped
+      - Dropped
+    * - 10.7
+      - from Oct 2010
+      - Ended
+      - Supported for the moment
+      - Dropped
+      - Dropped
+    * - 10.8
+      - from Feb 2012
+      - Supported for the moment
+      - Supported
+      - Supported
+      - Supported
+    * - 10.9
+      - from Jun 2013
+      - Supported
+      - Supported
+      - Supported
+      - Supported
+    * - 10.10
+      - from Oct 2014
+      - Supported
+      - Supported
+      - Recommended
+      - Recommended
+
+Apple don't formally announce end of life for their releases, but 10.6
+security fixes ended some time back, and 10.7 has now also ended with
+the release of 10.10.  We have regular CI testing of 10.8, 10.9 and
+10.10 builds plus developer testing of building and client and server
+deployment.  10.6 and 10.7 are marked as dropped since we no longer
+use 10.6 internally, and never used 10.7, and they are no longer
+tested by the CI infrastructure (all 10.6 and 10.7 nodes retired).
+
+UNIX (FreeBSD)
+^^^^^^^^^^^^^^
+
+It only really makes sense to support the base toolchain for major
+releases and the Ports tree (which is continually updated); these will
+be covered in the dependencies, below.
+
+Linux (Centos and RHEL)
+^^^^^^^^^^^^^^^^^^^^^^^
+
+General overview for `RHEL
+<https://access.redhat.com/site/articles/3078>`__ and `CentOS
+<http://wiki.centos.org/FAQ/General#head-fe8a0be91ee3e7dea812e8694491e1dde5b75e6d>`__
+
+.. tabularcolumns:: |l|l|l|l|l|J|
+
+.. list-table::
+    :header-rows: 1
+
+    * - Version
+      - Release date
+      - Upstream support
+      - OME (build)
+      - OME (deploy)
+      - Details
+    * - 5
+      - from Mar 2007
+      - to Mar 2017
+      - Dropped
+      - Dropped
+      - `Link <http://wiki.centos.org/FAQ/General#head-fe8a0be91ee3e7dea812e8694491e1dde5b75e6d>`__
+    * - 6
+      - from Nov 2010
+      - to Nov 2020
+      - Recommended
+      - Recommended
+      - `Link <http://wiki.centos.org/FAQ/General#head-fe8a0be91ee3e7dea812e8694491e1dde5b75e6d>`__
+    * - 7
+      - from June 2014
+      - to June 2024
+      - Upcoming
+      - Upcoming
+      - `Link <http://wiki.centos.org/FAQ/General#head-fe8a0be91ee3e7dea812e8694491e1dde5b75e6d>`__
+
+Only RHEL and CentOS 6 are supported at present.  Given the long life
+of enterprise releases, I'd suggest we should only support the latest
```

Comment:
That's intentional.  I just wrote that as a placeholder for having a concrete policy, which I hope can be decided on here or as a followup.  Same for the other comments/questions in the text.